tests/stubtest_whitelists/py38.txt

```diff
@@ -34,7 +34,6 @@ collections.abc.AsyncGenerator.ag_running
 collections.abc.ItemsView.__reversed__
 collections.abc.KeysView.__reversed__
 collections.abc.ValuesView.__reversed__
-concurrent.futures.ProcessPoolExecutor.map
```

Comment:
I wonder why this wasn't deleted last Saturday when `stubtest-unused-whitelist.yml` ran